modules/tinymce/changelog.txt

```diff
@@ -34,13 +34,13 @@ Version 5.7.0 (TBD)
     Fixed an issue where parsing malformed comments could cause an infinite loop #TINY-6864
     Fixed incorrect return types on `editor.selection.moveToBookmark` #TINY-6504
     Fixed the type signature for `editor.selection.setCursorLocation()` incorrectly allowing a node with no `offset` #TINY-6843
-    Fixed incorrect behaviour when editor is destroyed while loading stylesheets #INT-2282
+    Fixed incorrect behavior when editor is destroyed while loading stylesheets #INT-2282
     Fixed figure elements incorrectly splitting from a valid parent element when editing the image within #TINY-6592
     Fixed inserting multiple rows or columns in a table cloning from the incorrect source row or column #TINY-6906
-    Fixed an issue where new lines were not scrolled into view when pressing Shift+Enter #TINY-6964
-    Fixed an issue where list elements would not be removed when outdenting using enter #TINY-5974
+    Fixed an issue where new lines were not scrolled into view when pressing Shift+Enter or Shift+Return #TINY-6964
+    Fixed an issue where list elements would not be removed when outdenting using the Enter or Return key #TINY-5974
     Fixed an issue where file extensions with uppercase characters were treated as invalid #TINY-6940
-    Fixed dialog block messages unable to be translated #TINY-6971
+    Fixed dialog block messages were unable to be translated #TINY-6971
```

Comment:
This one reads weird still... `Fixed dialog block message could not be translated`? Not sure that's any better.

Reply:
@metricjs was it that they were hardcoded so we couldn't get them translated?

Reply:
The strings used in the `api.block(message)` call were never passed through the TinyMCE translation system. So not sure hard-coded is the right word, but it'd be equivalent.

Reply:
If not using one of our plugins, you could have technically done something like this to workaround it, which is why I said hard-coded wouldn't be right:
```
api.block(editor.translate(message))
```

Reply:
could we just do this?
```suggestion
    Fixed dialog block messages were not translated #TINY-6971
```
maybe implies the translations were missing instead, but it's not incorrect...

Reply:
Yeah, that definitely sounds like they are being translated now, which is wrong as we haven't got them translated yet.

Reply:
yeah... 
```suggestion
    Fixed dialog block messages were not passed through TinyMCE's translation system #TINY-6971
```
just be explicit?